docs/sources/panels-visualizations/visualizations/flame-graph/index.md

```diff
@@ -16,57 +16,85 @@ title: Flame graph
 weight: 850
 ---
 
-# Flame graph panel
+# Flame graph
 
-The flame graph takes advantage of the hierarchical nature of profiling data. It condenses data into a format that allows you to easily see which code paths are consuming the most system resources.
+Flame graph panel is a visualization that allows you to visualize profiling data. It can represent the profile as a flame graph, table, or both
 
-These resources are measured through profiles which aggregate that information into a format which is then sent to the flame graph visualization. For example, allocated objects or space when measuring memory.
+![<Flame graph panel>](/media/docs/grafana/panels-visualizations/flamegraph/screenshot-flamegraph-10.1.png)
 
-![Figure 1 - Flame graph](/static/img/docs/flame-graph-panel/flame-graph.png 'Figure 1 - Flame graph')
+## Flame graph mode
 
-Figure 1 illustrates what this data looks like when rendered. The top block is the root or parent of the other blocks, and represents the total of all of the profiles being measured. There may be one or more blocks for each row that is rendered below the total. Each of these blocks represents a child in the stack of function calls that originated from the original call to the root at the top of the flame graph.
+The flame graph takes advantage of the hierarchical nature of profiling data. It condenses data into a format that allows you to easily see which code paths are consuming the most system resources, for example, CPU time, allocated objects or space when measuring memory. Each block in the flame graph represents a function call in a stack and its width represents its value.
 
-Each block can have zero or more siblings but can only have one parent.
+![<Flame graph>](/media/docs/grafana/panels-visualizations/flamegraph/screenshot-flamegraph-10.1-flamegraph.png)
 
-In figure 1 displays many rows of colored blocks as well as grayed-out sections which are a set of blocks that represent a relatively short execution time and were collapsed together into one section for performance reasons.
+Grayed-out sections are a set of functions that represent a relatively small value and they've been collapsed together into one section for performance reasons.
```

Comment:
```suggestion
Grayed-out sections are a set of functions that represent a relatively small value and they are collapsed together into one section for performance reasons.
```

Reply:
Yeah they are kinda purple aren't they? :D I think it's mainly because we gray them in the sense of desaturating some color so they are more "gray" than other parts.

Reply:
Would like to avoid using specific color names just in case we change it to really gray or have a bit different color for dark/light theme but wonder what word would describe this, desaturated? Gray-ish? I feel like grayed-out could make sense here overall as a "comparatively gray" to other parts of the graph

Reply:
To be fair, I only got confused when I was reading it for the third time and when I first read it, it made sense to me. So this may just be me looking at it too long. Let's leave it for the moment.